src/game/collide.c

```diff
@@ -40,10 +43,27 @@ void Collide_GetCollisionInfo(
     coll->mid_ceiling = ceiling;
     coll->mid_type = g_HeightType;
     coll->trigger = g_TriggerIndex;
-
-    int16_t tilt = Room_GetTiltType(floor, x, g_LaraItem->pos.y, z);
-    coll->tilt_z = tilt >> 8;
-    coll->tilt_x = (int8_t)tilt;
+    bool skip_slope_check = false;
+
+    // TODO Do we need to check all walkable heights or is knowing Lara is above
+    // one bridge enough? Can a slope ever be below one bridge but above
+    // another?
```

Comment:
I don't think this is possible unless a level builder embeds an object in the room geometry, which serves no purpose. Or they embed the bridge in a way so that it protrudes from a slope so that it uses mixed room/object collision, but it's unreasonable to expect that to work IMO.
So all in all I think it's good to remove this TODO.